pom.xml

```diff
@@ -63,6 +63,11 @@
             <artifactId>gherkin</artifactId>
             <version>${gherkin.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.15.2</version>
+        </dependency>
```

Comment:
it is only used for tests, so it should have the test scope. And, BTW: jaxb can be dropped as well